.gitlab-ci.yml

```diff
@@ -56,18 +56,31 @@ validate-log-integrations:
 trigger-agent-build:
   stage: trigger
   image: $VALIDATE_AGENT_BUILD
+  # Exclude release branches
+  workflow:
+    rules:
+      - if: ($CI_COMMIT_BRANCH !~ /^7.[0-9]{2}.x/)
+        changes:
+          - datadog_checks_base/datadog_checks/base/data/agent_requirements.in
+        when: never
+      - when: on_success
```

Comment:
The `trigger-agent-build`  pipeline is ignored on release branches (branches looking like `7.YY.x`) when `agent_requirements.in` is not changed.
So, cherry-picked commit on release branch most of the time WON'T trigger agent build